oeps/oep-0047-bp-semantic-versioning.rst

```diff
@@ -0,0 +1,90 @@
+###########################
+OEP-47: Semantic Versioning
+###########################
+
+.. list-table::
+
+   * - OEP
+     - :doc:`OEP-0047 </oeps/oep-0047>`
+   * - Title
+     - Semantic Versioning
+   * - Last Modified
+     - 2020-05-08
+   * - Authors
+     - Robert Raposa <rraposa@edx.org>
+   * - Arbiter
+     - Régis Behmo <regis@behmo.com>
+   * - Status
+     - Under Review
+   * - Type
+     - Best Practice
+   * - Created
+     - 2020-03-19
+   * - Review Period
+     - 2020-05-11 - 2020-05-25
+
+.. contents::
+   :local:
+   :depth: 3
+
+Summary
+=======
+
+For any Open edX repository that publishes versioned packages, this OEP recommends following the `Semantic Versioning v2`_.
```

Comment:
We sometimes maintain forks of other projects. Should we include something about versioning those? (Stick with their scheme, use our own, use a `X.Y.Z+edx1` build indicator, etc.)

Reply:
Adding a `+edx.1` suffix to the version number is now the standard way to handle this, after the Python community considered all the options: https://www.python.org/dev/peps/pep-0440/#local-version-identifiers .

Reply:
Added a section.